guides/admin/wp-cli.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Workspace provider provisioning via CLI
+description: Learn how to provision a workspace provider using the Coder CLI.
+---
+
+1. Install and authenticate the Coder CLI.
+
+1. Run the following to provision a new workspace provider (be sure to replace
+   the placeholders as necessary):
+
+   ```console
+   coder providers create [name] --hostname=[hostname] --cluster-address=[clusterAddress]
```

Comment:
This looks like it is creating a Kubernetes provider so we need to add `kubernetes` sub-command.  Also, I think the `--hostname` flag does not exist.  The minimum required flags are `--cluster-address` and `--namespace` so we could use those.

```suggestion
   coder providers create kubernetes [name] --namespace=[namespace] --cluster-address=[clusterAddress]
```

Reply:
One thought is that  we could have both a Kubernetes section and an EC2 section.  The minimal EC2 command is:

``` 
coder providers create ec2 [name] --access-key-id=[access-key-id] --secret-access-key=[secret-access-key]
```